app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt

```diff
@@ -0,0 +1,21 @@
+package de.tum.`in`.tumcampusapp.component.tumui.calendar
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.webkit.WebView
+
+class NavigatumActivity : AppCompatActivity() {
```

Comment:
```suggestion
class NavigaTUMActivity : AppCompatActivity() {
```

Reply:
`app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt` could be changed also, if you decide that these suggenstions are valid